<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1041</width>
    <height>568</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="windowTitle">
   <string>Generadores</string>
  </property>
  <property name="windowIcon">
   <iconset resource="resource.qrc">
    <normaloff>:/images/images/dice.png</normaloff>:/images/images/dice.png</iconset>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QWidget" name="horizontalLayoutWidget">
    <property name="geometry">
     <rect>
      <x>9</x>
      <y>9</y>
      <width>1021</width>
      <height>531</height>
     </rect>
    </property>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <layout class="QVBoxLayout" name="verticalLayout_7">
       <item>
        <widget class="QTabWidget" name="tabWidget">
         <property name="font">
          <font>
           <pointsize>12</pointsize>
          </font>
         </property>
         <property name="currentIndex">
          <number>0</number>
         </property>
         <widget class="QWidget" name="tab_1">
          <attribute name="title">
           <string>Cuadrados Medios</string>
          </attribute>
          <widget class="QWidget" name="formLayoutWidget_2">
           <property name="geometry">
            <rect>
             <x>10</x>
             <y>10</y>
             <width>481</width>
             <height>161</height>
            </rect>
           </property>
           <layout class="QFormLayout" name="formLayout_2">
            <item row="1" column="0">
             <widget class="QLabel" name="label_6">
              <property name="font">
               <font>
                <pointsize>12</pointsize>
               </font>
              </property>
              <property name="text">
               <string>n</string>
              </property>
             </widget>
            </item>
            <item row="1" column="1">
             <widget class="QSpinBox" name="spinBox_CM_n">
              <property name="minimum">
               <number>1</number>
              </property>
              <property name="maximum">
               <number>100</number>
              </property>
             </widget>
            </item>
            <item row="2" column="0">
             <widget class="QLabel" name="label_7">
              <property name="text">
               <string>X0</string>
              </property>
             </widget>
            </item>
            <item row="2" column="1">
             <widget class="QSpinBox" name="spinBox_CM_x0">
              <property name="minimum">
               <number>1111</number>
              </property>
              <property name="maximum">
               <number>9999</number>
              </property>
              <property name="value">
               <number>1111</number>
              </property>
             </widget>
            </item>
           </layout>
          </widget>
         </widget>
         <widget class="QWidget" name="tab_2">
          <attribute name="title">
           <string>Productos Medios</string>
          </attribute>
          <widget class="QWidget" name="formLayoutWidget_3">
           <property name="geometry">
            <rect>
             <x>10</x>
             <y>10</y>
             <width>481</width>
             <height>161</height>
            </rect>
           </property>
           <layout class="QFormLayout" name="formLayout_3">
            <item row="1" column="0">
             <widget class="QLabel" name="label_8">
              <property name="font">
               <font>
                <pointsize>12</pointsize>
               </font>
              </property>
              <property name="text">
               <string>n</string>
              </property>
             </widget>
            </item>
            <item row="1" column="1">
             <widget class="QSpinBox" name="spinBox_PM_n">
              <property name="minimum">
               <number>1</number>
              </property>
              <property name="maximum">
               <number>100</number>
              </property>
             </widget>
            </item>
            <item row="2" column="0">
             <widget class="QLabel" name="label_9">
              <property name="text">
               <string>X0</string>
              </property>
             </widget>
            </item>
            <item row="2" column="1">
             <widget class="QSpinBox" name="spinBox_PM_x0">
              <property name="minimum">
               <number>1111</number>
              </property>
              <property name="maximum">
               <number>9999</number>
              </property>
              <property name="value">
               <number>1111</number>
              </property>
             </widget>
            </item>
            <item row="3" column="0">
             <widget class="QLabel" name="label_10">
              <property name="text">
               <string>X1</string>
              </property>
             </widget>
            </item>
            <item row="3" column="1">
             <widget class="QSpinBox" name="spinBox_PM_x1">
              <property name="minimum">
               <number>1111</number>
              </property>
              <property name="maximum">
               <number>9999</number>
              </property>
              <property name="value">
               <number>1111</number>
              </property>
             </widget>
            </item>
           </layout>
          </widget>
         </widget>
         <widget class="QWidget" name="tab">
          <attribute name="title">
           <string>Multiplicar Constante</string>
          </attribute>
          <widget class="QWidget" name="formLayoutWidget_4">
           <property name="geometry">
            <rect>
             <x>10</x>
             <y>10</y>
             <width>481</width>
             <height>161</height>
            </rect>
           </property>
           <layout class="QFormLayout" name="formLayout_4">
            <item row="1" column="0">
             <widget class="QLabel" name="label_11">
              <property name="font">
               <font>
                <pointsize>12</pointsize>
               </font>
              </property>
              <property name="text">
               <string>n</string>
              </property>
             </widget>
            </item>
            <item row="1" column="1">
             <widget class="QSpinBox" name="spinBox_MC_n">
              <property name="minimum">
               <number>1</number>
              </property>
              <property name="maximum">
               <number>1000</number>
              </property>
             </widget>
            </item>
            <item row="2" column="0">
             <widget class="QLabel" name="label_12">
              <property name="text">
               <string>X0</string>
              </property>
             </widget>
            </item>
            <item row="2" column="1">
             <widget class="QSpinBox" name="spinBox_MC_x0">
              <property name="minimum">
               <number>1111</number>
              </property>
              <property name="maximum">
               <number>9999</number>
              </property>
              <property name="value">
               <number>1111</number>
              </property>
             </widget>
            </item>
            <item row="3" column="0">
             <widget class="QLabel" name="label_14">
              <property name="text">
               <string>c</string>
              </property>
             </widget>
            </item>
            <item row="3" column="1">
             <widget class="QSpinBox" name="spinBox_MC_c">
              <property name="minimum">
               <number>1111</number>
              </property>
              <property name="maximum">
               <number>9999</number>
              </property>
              <property name="value">
               <number>1111</number>
              </property>
             </widget>
            </item>
           </layout>
          </widget>
         </widget>
         <widget class="QWidget" name="tab_3">
          <attribute name="title">
           <string>Algoritmo Lineal</string>
          </attribute>
          <widget class="QWidget" name="formLayoutWidget">
           <property name="geometry">
            <rect>
             <x>10</x>
             <y>10</y>
             <width>481</width>
             <height>161</height>
            </rect>
           </property>
           <layout class="QFormLayout" name="formLayout">
            <item row="1" column="0">
             <widget class="QLabel" name="label">
              <property name="font">
               <font>
                <pointsize>12</pointsize>
               </font>
              </property>
              <property name="text">
               <string>n</string>
              </property>
             </widget>
            </item>
            <item row="2" column="0">
             <widget class="QLabel" name="label_2">
              <property name="text">
               <string>X0</string>
              </property>
             </widget>
            </item>
            <item row="3" column="0">
             <widget class="QLabel" name="label_3">
              <property name="text">
               <string>a</string>
              </property>
             </widget>
            </item>
            <item row="4" column="0">
             <widget class="QLabel" name="label_4">
              <property name="text">
               <string>c</string>
              </property>
             </widget>
            </item>
            <item row="5" column="0">
             <widget class="QLabel" name="label_5">
              <property name="text">
               <string>m</string>
              </property>
             </widget>
            </item>
            <item row="1" column="1">
             <widget class="QSpinBox" name="spinBox_AL_n">
              <property name="minimum">
               <number>1</number>
              </property>
              <property name="maximum">
               <number>100</number>
              </property>
             </widget>
            </item>
            <item row="5" column="1">
             <widget class="QSpinBox" name="spinBox_AL_m">
              <property name="minimum">
               <number>1</number>
              </property>
              <property name="maximum">
               <number>999</number>
              </property>
              <property name="value">
               <number>1</number>
              </property>
             </widget>
            </item>
            <item row="4" column="1">
             <widget class="QSpinBox" name="spinBox_AL_c">
              <property name="minimum">
               <number>1</number>
              </property>
              <property name="maximum">
               <number>100</number>
              </property>
              <property name="value">
               <number>1</number>
              </property>
             </widget>
            </item>
            <item row="2" column="1">
             <widget class="QSpinBox" name="spinBox_AL_x0">
              <property name="minimum">
               <number>1</number>
              </property>
              <property name="maximum">
               <number>100</number>
              </property>
              <property name="value">
               <number>1</number>
              </property>
             </widget>
            </item>
            <item row="3" column="1">
             <widget class="QSpinBox" name="spinBox_AL_a">
              <property name="minimum">
               <number>1</number>
              </property>
              <property name="maximum">
               <number>100</number>
              </property>
              <property name="value">
               <number>1</number>
              </property>
             </widget>
            </item>
           </layout>
          </widget>
         </widget>
         <widget class="QWidget" name="tab_4">
          <attribute name="title">
           <string>Algoritmo Congruencial Multiplicativo</string>
          </attribute>
          <widget class="QWidget" name="formLayoutWidget_5">
           <property name="geometry">
            <rect>
             <x>10</x>
             <y>10</y>
             <width>481</width>
             <height>182</height>
            </rect>
           </property>
           <layout class="QFormLayout" name="formLayout_5">
            <item row="1" column="0">
             <widget class="QLabel" name="label_15">
              <property name="text">
               <string>X0</string>
              </property>
             </widget>
            </item>
            <item row="1" column="1">
             <widget class="QSpinBox" name="spinBox_ACM_x0">
              <property name="minimum">
               <number>1</number>
              </property>
              <property name="maximum">
               <number>9999</number>
              </property>
              <property name="value">
               <number>1</number>
              </property>
             </widget>
            </item>
            <item row="2" column="0">
             <widget class="QLabel" name="label_16">
              <property name="text">
               <string>k</string>
              </property>
             </widget>
            </item>
            <item row="2" column="1">
             <widget class="QSpinBox" name="spinBox_ACM_k">
              <property name="minimum">
               <number>1</number>
              </property>
              <property name="maximum">
               <number>100</number>
              </property>
              <property name="value">
               <number>1</number>
              </property>
             </widget>
            </item>
            <item row="3" column="0">
             <widget class="QLabel" name="label_18">
              <property name="text">
               <string>m</string>
              </property>
             </widget>
            </item>
            <item row="3" column="1">
             <widget class="QSpinBox" name="spinBox_ACM_m">
              <property name="minimum">
               <number>1</number>
              </property>
              <property name="maximum">
               <number>999</number>
              </property>
              <property name="value">
               <number>1</number>
              </property>
             </widget>
            </item>
           </layout>
          </widget>
         </widget>
         <widget class="QWidget" name="tab_5">
          <attribute name="title">
           <string>Algoritmo Congruencial Aditivo</string>
          </attribute>
          <widget class="QWidget" name="formLayoutWidget_6">
           <property name="geometry">
            <rect>
             <x>10</x>
             <y>10</y>
             <width>481</width>
             <height>71</height>
            </rect>
           </property>
           <layout class="QFormLayout" name="formLayout_6">
            <item row="1" column="0">
             <widget class="QLabel" name="label_20">
              <property name="text">
               <string>Xi</string>
              </property>
             </widget>
            </item>
            <item row="1" column="1">
             <widget class="QSpinBox" name="spinBox_ACA_xi">
              <property name="minimum">
               <number>1</number>
              </property>
              <property name="maximum">
               <number>9999</number>
              </property>
              <property name="value">
               <number>1</number>
              </property>
             </widget>
            </item>
            <item row="2" column="0">
             <widget class="QLabel" name="label_23">
              <property name="text">
               <string>m</string>
              </property>
             </widget>
            </item>
            <item row="2" column="1">
             <widget class="QSpinBox" name="spinBox_ACA_m">
              <property name="minimum">
               <number>1</number>
              </property>
              <property name="maximum">
               <number>999</number>
              </property>
              <property name="value">
               <number>1</number>
              </property>
             </widget>
            </item>
           </layout>
          </widget>
          <widget class="QListWidget" name="listWidget_ACA_Numeros">
           <property name="geometry">
            <rect>
             <x>10</x>
             <y>130</y>
             <width>481</width>
             <height>211</height>
            </rect>
           </property>
           <property name="font">
            <font>
             <pointsize>12</pointsize>
            </font>
           </property>
          </widget>
          <widget class="QLabel" name="label_21">
           <property name="geometry">
            <rect>
             <x>170</x>
             <y>100</y>
             <width>161</width>
             <height>25</height>
            </rect>
           </property>
           <property name="text">
            <string>Conjunto de numeros</string>
           </property>
          </widget>
          <widget class="QPushButton" name="pushButton_ACA_Agregar">
           <property name="geometry">
            <rect>
             <x>330</x>
             <y>350</y>
             <width>75</width>
             <height>31</height>
            </rect>
           </property>
           <property name="text">
            <string>Agregar</string>
           </property>
          </widget>
          <widget class="QPushButton" name="pushButton_ACA_Eliminar">
           <property name="geometry">
            <rect>
             <x>410</x>
             <y>350</y>
             <width>75</width>
             <height>31</height>
            </rect>
           </property>
           <property name="text">
            <string>Eliminar</string>
           </property>
          </widget>
          <widget class="QSpinBox" name="spinBox_ACA_Agregar">
           <property name="geometry">
            <rect>
             <x>10</x>
             <y>350</y>
             <width>311</width>
             <height>31</height>
            </rect>
           </property>
           <property name="minimum">
            <number>1</number>
           </property>
           <property name="maximum">
            <number>1000</number>
           </property>
           <property name="value">
            <number>1</number>
           </property>
          </widget>
         </widget>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="pushButton_Calcular">
         <property name="font">
          <font>
           <pointsize>12</pointsize>
          </font>
         </property>
         <property name="text">
          <string>Calcular</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="pushButton_BorrarDatos">
         <property name="font">
          <font>
           <pointsize>12</pointsize>
          </font>
         </property>
         <property name="text">
          <string>Borrar Datos</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="pushButton">
         <property name="font">
          <font>
           <pointsize>12</pointsize>
          </font>
         </property>
         <property name="text">
          <string>Borrar Numeros Aleatorios</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QVBoxLayout" name="verticalLayout_4">
       <item>
        <widget class="QListWidget" name="listWidget_R">
         <property name="font">
          <font>
           <pointsize>14</pointsize>
          </font>
         </property>
        </widget>
       </item>
      </layout>
     </item>
    </layout>
   </widget>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources>
  <include location="resource.qrc"/>
 </resources>
 <connections>
  <connection>
   <sender>pushButton</sender>
   <signal>clicked()</signal>
   <receiver>listWidget_R</receiver>
   <slot>clear()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>225</x>
     <y>511</y>
    </hint>
    <hint type="destinationlabel">
     <x>692</x>
     <y>370</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
